src/Dapr.Actors/Runtime/ReminderInfo.cs

```diff
@@ -26,12 +26,14 @@ internal struct ReminderInfo
             byte[] data,
             TimeSpan dueTime,
             TimeSpan period,
+            int? repetitions,
```

Comment:
Can this not be default valued? I know we can set it as null but we are forcing the set.

Reply:
Fixed